sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/_additional_includes.py

```diff
@@ -40,7 +40,7 @@ class AdditionalIncludes:
     def __init__(
         self,
         *,
-        origin_code_value: Union[None, str],
+        origin_code_value: Optional[str],
```

Comment:
why is it originally like: Union[None, str]?

Reply:
I guess no special reason. It's brought in by a commit by ying 4 months ago @YingChen1996 please correct me if there is some specific reason